src/test/java/uk/gov/pay/adminusers/service/UserServicesTest.java

```diff
@@ -259,12 +284,12 @@ public void shouldReturnUser_whenIncrementingSessionVersion_ifUserFound() throws
         UserEntity userEntity = aUserEntityWithTrimmings(user);
 
         Optional<UserEntity> userEntityOptional = Optional.of(userEntity);
-        when(userDao.findByUsername("random-name")).thenReturn(userEntityOptional);
+        when(userDao.findByUsername(USER_USERNAME)).thenReturn(userEntityOptional);
 
-        Optional<User> userOptional = userServices.patchUser("random-name", PatchRequest.from(node));
+        Optional<User> userOptional = userServices.patchUser(USER_USERNAME, PatchRequest.from(node));
```

Comment:
We should possibly start adopting EXTERNAL_IDs for these methods (instead of USER_USERNAME). But can be done in a next PR if needed.